xapian-applications/omega/templates/query

```diff
@@ -44,7 +44,7 @@ function write_date(time_t) {
 var d=new Date();d.setTime(time_t*1000);
 var D=d.getFullYear()+"-"+f2(d.getMonth()+1)+"-"+f2(d.getDate());
 var T=f2(d.getHours())+":"+f2(d.getMinutes())+":"+f2(d.getSeconds());
-document.write("<span title=\""+D+" "+T+"\">Modified:<b>"+D+"</b></span>");
+document.write("<span>Modified:<time datetime=\""+D+" "+T+"\"><b>"+D+"</b></time></span>");
```

Comment:
If we're going to keep a `span` in there, it should have a class to distinguish it. Also, I suggested keeping the `title` but on `time` so the hover state works, and additionally I pointed out that the syntax for `datetime` is different to that used for the `title` attribute previously.